src/components/utils/SelectSpacePreview.tsx

```diff
@@ -29,19 +31,20 @@ const SpaceOption = ({ spaceId, imageSize }: SpaceOptionProps) => {
 
   const { struct, content } = spaceStruct
 
-  if (!content) return spaceStub
+  const name = content?.name
 
   return (
     <div className={'d-flex align-items-center'}>
       <SpaceAvatar
-        address={struct.ownerId}
         space={struct}
         avatar={content?.image}
         size={imageSize}
+        isUnnamedSpace={isEmptyStr(name) ? true : false}
         asLink={false}
       />
       <div className='content'>
-        <div className='handle'>{content?.name}</div>
+        {!content && <MutedDiv>{'Unnamed Space'}</MutedDiv>}
```

Comment:
change this to !content?.name, because in previous one, you use this condition to display this name instead of just !content